src/main/java/seedu/address/MainApp.java

```diff
@@ -15,26 +15,8 @@
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.Logic;
 import seedu.address.logic.LogicManager;
-import seedu.address.model.ContactList;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.ModuleList;
-import seedu.address.model.ReadOnlyContactList;
-import seedu.address.model.ReadOnlyModuleList;
-import seedu.address.model.ReadOnlyTodoList;
-import seedu.address.model.ReadOnlyUserPrefs;
-import seedu.address.model.TodoList;
-import seedu.address.model.UserPrefs;
-import seedu.address.storage.ContactListStorage;
-import seedu.address.storage.JsonContactListStorage;
-import seedu.address.storage.JsonModuleListStorage;
-import seedu.address.storage.JsonTodoListStorage;
-import seedu.address.storage.JsonUserPrefsStorage;
-import seedu.address.storage.ModuleListStorage;
-import seedu.address.storage.Storage;
-import seedu.address.storage.StorageManager;
-import seedu.address.storage.TodoListStorage;
-import seedu.address.storage.UserPrefsStorage;
+import seedu.address.model.*;
+import seedu.address.storage.*;
```

Comment:
Should expand to single line imports to pass CI checks

Reply:
Done! Thanks!